wfl/Controllers/FoodOrdersController.cs

```diff
@@ -68,5 +68,19 @@ public FoodOrdersController(IDBManager dbManager)
             order.setStatus(EntityStatus.Status.Active);
             return Ok(await dbManager.UpdateEntityAsync(id, order));
         }
+
+        [Route("{id}/complete")]
+        [HttpPost]
+        public async Task<IHttpActionResult> CompleteAsync([FromUri] string id)
```

Comment:
I think @jspringer2020 mentioned this in another PR, but we could follow suit here and make this a generic controller method to set the status. The interesting part comes in the profile. There you advertise certain actions based on what the current status is. So IF an orders status is `Active` the profile will show a `complete` action. That action will call this method with the status `Completed`.